osf/migrations/0155_add_registration_files_count.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2019-01-18 14:43
+from __future__ import unicode_literals
+import logging
+
+from django.db import migrations, models
+from django_bulk_update.helper import bulk_update
+
+logger = logging.getLogger(__file__)
+
+def add_registration_files_count(state, *args, **kwargs):
+    AbstractNode = state.get_model('osf.abstractnode')
+    BaseFileNode = state.get_model('osf.basefilenode')
+    ContentType = state.get_model('contenttypes', 'ContentType')
+    registrations = AbstractNode.objects.filter(type='osf.registration', is_deleted=False)
+    content_type = ContentType.objects.get(app_label='osf', model='abstractnode')
+
+    registrations_to_update = []
+    for registration in registrations:
+        job = registration.archive_jobs.first() if registration.archive_jobs.count() else None
```

Comment:
```suggestion
        job = registration.archive_job
```
or is it a problem to depend on model properties in a migration?

Reply:
Correct. See [historical models](https://docs.djangoproject.com/en/2.1/topics/migrations/#historical-models).